helo.java

```diff
@@ -0,0 +1,5 @@
+class Main {
```

Comment:
`public class`

Reply:
i might

Reply:
ok i did it https://cdn.discordapp.com/emojis/1100919837414395904.gif?size=128&quality=lossless